src/ORM/Table.php

```diff
@@ -1436,6 +1435,36 @@ public function get($primaryKey, $options = [])
     }
 
     /**
+     * Handles the logic executing of a worker inside a transaction.
+     *
+     * @param bool $atomic Whether to execute the worker inside a database transaction.
+     * @param callable $worker The worker that will run inside the transaction.
+     * @return mixed
+     */
+    protected function _executeTransaction($atomic, callable $worker)
```

Comment:
@lorenzo But isn't having similarity with PHP functions better?

Also i don't really dig the method name `_executeTransaction()` given the fact it won't do a transaction if `$atomic` is false.